packages/core/test/controls/numericInputTests.tsx

```diff
@@ -873,9 +873,11 @@ describe("<NumericInput>", () => {
         it("shows a left icon if provided", () => {
             const component = mount(<NumericInput leftIconName={"variable"} />);
 
-            const inputGroup = component.find(InputGroup);
-            const icon = inputGroup.childAt(0);
-
+            const icon = component
+                .find(InputGroup)
+                .children()
+                .childAt(0)
+                .childAt(0);
```

Comment:
extra `childAt` to get the `span` inside `Icon`

Reply:
leave a comment to explain this, e.g.

```
.childAt(0) // Icon
.childAt(0) // span
```